root/usr/share/fdi/facts/discovery-facts.rb

```diff
@@ -18,23 +18,30 @@
 # also available at http://www.gnu.org/copyleft/gpl.html.
 
 require 'resolv'
-require 'facter/util/ip'
-require '/usr/lib64/ruby/vendor_ruby/discovery.rb'
+if File.exist? '/opt/rh/rh-ruby25/root/usr/share/ruby/vendor_ruby/discovery'
+  require '/opt/rh/rh-ruby25/root/usr/share/ruby/vendor_ruby/discovery'
+else
+  require_relative '../../../../opt/rh/rh-ruby25/root/usr/share/ruby/vendor_ruby/discovery.rb'
```

Comment:
This feels odd, what would be the absolute variant of this?

Reply:
This is only used when running this from git, no big deal. It would require users to modify LOAD_PATH otherwise.